modules/client-configuration/pages/clients-centos.adoc

```diff
@@ -176,6 +176,14 @@ endif::[]
 For more information, see xref:client-configuration:registration-overview.adoc[].
 
 
+[WARNING]
+====
+Some clients experience SSL errors when you register them in the {productname} {webui} using a bootstrap script.
```

Comment:
Not some, but CentOS6 also I find "when you register them in the {productname} {webui} using a bootstrap script" confusing. You just "register" systems on {productname} and you can do that from WebUI or using a bootstrap script (or IIRC also from API).